lib/rester.rb

```diff
@@ -19,6 +20,11 @@ def load_tasks
     def connect(*args)
       if (service = args.first).is_a?(Class) && service < Service
         Client.new(Client::Adapters::LocalAdapter.new(*args))
+      elsif args.first.is_a?(String) && Pathname(args.first).file?
+        unless (file_ext = Pathname(args.first).extname) == '.yml'
+          raise Errors::InvalidStubFileError, "Expected .yml, got #{file_ext}"
+        end
+        Client.new(Client::Adapters::StubAdapter.new(*args))
```

Comment:
Let's change this method do be more explicit instead of loose wrapper around Client.

For example:
```ruby
def connect(service, params={})
  if service.is_a?(Class) && service < Service
    adapter = Client::Adapters::LocalAdapter.new(service)
  elsif service.is_a?(String) && Pathname(service).file?
    adapter = Client::Adapters::StubAdapter.new(service)
  elsif service.is_a?(String) && ['http', 'https'].include?(URI(service).scheme)
    adapter = Client::Adapters::HttpAdapter.new(service)
  else
    fail "unable to connect to: #{service.inspect}"
  end

  Client.new(adapter, params)
end
```